src/contexts/VoiceRecorderContext.tsx

```
@@ -0,0 +1,71 @@
+
+// Provider
+export const VoiceRecorderProvider: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => {
+  const [transcript, setTranscript] = useState("");
+
+  useEffect(() => {
+    // Override console.warn to filter onnxruntime warnings globally
+    const originalWarn = console.warn;
+    console.warn = (message, ...args) => {
+      if (message && message.includes("onnxruntime")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)